### DISCUSSION TITLE
In the inverse problem solving networks of scirun software, the forward matrix is the mapping from epicardium to body surface or body surface sensor?

### DISCUSSION THREAD

**Discussion by zhifeiji-han** [SCIInstitute/SCIRun#2314] [General]
I've looked at all the inverse problem programs based on potential.

In tikhonov-inverse.srn5, Tikhonov svd-inverse.srn5, tsvd-inverse.srn5, greensite-inverse-isotropy method.srn5, the dimension of the forward solution matrix is 771 * 128.
In WBPR.srn5, the dimension of forward solution matrix is 300 * 257.
In spline-inverse.srn5, the dimension of the forward solution matrix is 114 * 350.
In the inverse problem solving networks of scirun software, the forward matrix is the mapping from epicardium to body surface or body surface sensor?
When I plot the field data of 771, it's a trunk with evenly distributed nodes. I think it's a trunk.
![image](https://user-images.githubusercontent.com/70640243/125017458-c5577180-e0a5-11eb-9e1b-378269e9db88.png)
I plot the field data of 300. The nodes are unevenly distributed. I think it should be the sensor array.
![image](https://user-images.githubusercontent.com/70640243/125017478-ce484300-e0a5-11eb-9f80-b5f1c1d53db0.png)
However, this is my guess. I'm not sure, and I don't know why it's set like this. I'm curious why it's different.

**Comment by zhifeiji-han** [comment]
@bryanworthen
@rsmacleod
@bzenger
@jab0707
@ jessdtate

**Comment by jab0707** [comment]
The mapping matrix, via the BuildBEMatrix, is from surface to surface. If your torso surface has more nodes than sensors then you would select only the rows corresponding to hose measurement locations when performing the inverse estimation. I think part of the problem with the networks you have highlighted is that they utilize different torso meshes, and thus have different node numbers and geometries.
